pom.xml

```diff
@@ -17,7 +17,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>co.cask.re</groupId>
+  <groupId>io.cdap.re</groupId>
   <artifactId>dre</artifactId>
   <version>1.2.0</version>
```

Comment:
This should change as we are making modifications.

Reply:
We haven't released/published dre 1.2.0 yet, so I think this is fine.

Reply:
yea I think for this specific case its ok to merge to non-snapshot version